File: BP10_Writeoff/src/main/java/com/lenta/bp10writeoff/MainActivity.java
package com.lenta.bp10writeoff;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;

import com.lenta.shared.DialogBuilder;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    public void imgBtn_onClick(View v)
    {
        // определяем нажатую кнопку и выполняем соответствующую операцию
        switch (v.getId()) {
            case R.id.ExitBtnImg:
                DialogBuilder dialogCloseApp = new DialogBuilder(this);
                dialogCloseApp.setTypeDialog(dialogCloseApp.TD_CLOSE_APP).setTitleText(R.string.name_buisness_process).show();
                break;
            case R.id.AuxiliaryMenuBtnImg:
                Intent intAuxiliaryMenu = new Intent(this, AuxiliaryMenuActivity.class);
                intAuxiliaryMenu.putExtra("MainActivity",true);
                startActivity(intAuxiliaryMenu);
                break;
            case R.id.CreateTaskBtnImg:
                Intent intTaskCard = new Intent(this, TaskCardActivity.class);
                startActivity(intTaskCard);
                break;
            default:
                break;
        }
    }
}

File: writeoff_breakage/src/main/java/com/lenta/bp10/MainActivity.kt
package com.lenta.bp10

import android.content.SharedPreferences
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import com.lenta.bp10.di.getAppComponent
import javax.inject.Inject

class MainActivity : AppCompatActivity() {

    @Inject
    lateinit var sharedPrefferences: SharedPreferences

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        getAppComponent().inject(this)


    }
}
